modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java

```diff
@@ -243,6 +243,13 @@ public List<ObjectField> getActiveObjectFields(
 		return activeObjectFields;
 	}
 
+	@Override
+	public List<ObjectField> getCustomObjectFields(
```

Comment:
I'm not sure about this method name, because the method name gives the impression that it will get CUSTOM fields but we are giving a choice of getting the SYSTEM fields by the parameter "system". I would rename for something more generic like "getObjectFields". What is your opinion about this? Do you have any name in mind?